src/lib/components/VideoCard/VideoCard.svelte

```diff
@@ -37,43 +37,55 @@
 
   $: title = customTitle;
   $: description = customDescription;
+  $: loadSubscribeButton = false;
 
   afterUpdate(async () => {
     if (youtubeVideoId && (!customTitle || !customDescription)) {
       const snippet = await getYoutubeVideoData(youtubeVideoId);
-      title = snippet.title;
-      description = snippet.description;
+      if (!customTitle) {
+        title = snippet.title;
+      }
+      if (!customDescription) {
+        // Only use the first paragraph instead of rendering with the full
+        //   description to avoid trimming at a newline (leaving an orphaned
+        //   ellipsis) and to hopefully avoid displaying URLs and hashtags.
+        description = snippet.description.split("\n")[0];
+      }
     }
+    loadSubscribeButton = true;
   });
 </script>
 
 {#if youtubeVideoId}
-  {#key youtubeVideoId}
-    <div class="ldaf-video-card ldaf-video-card--{variation} {className}">
-      <div class="ldaf-video-container">
-        <iframe
-          class="ldaf-video-embed"
-          src={`https://www.youtube-nocookie.com/embed/${youtubeVideoId}`}
-          title="Embedded YouTube video player"
-          frameborder="0"
-          allowfullscreen
+  <div class="ldaf-video-card ldaf-video-card--{variation} {className}">
+    <div class="ldaf-video-container">
+      <iframe
+        class="ldaf-video-embed"
+        src={`https://www.youtube-nocookie.com/embed/${youtubeVideoId}?rel=0`}
```

Comment:
Addition of the `rel=0` makes it so that YouTube only recommends other videos by LDAF when the user pauses.